nautobot/project-static/css/base.css

```diff
@@ -44,6 +45,7 @@ body {
     transition-property: margin-left, width;
     overflow: visible;
     margin: 0;
+    padding-right: 70px;
```

Comment:
This seems odd/unnecessary?

![image](https://github.com/nautobot/nautobot/assets/5603551/00329455-bd51-4104-9b0d-1650b5bb7f39)

Reply:
I still think adjusting the sizes of this column (https://github.com/nautobot/nautobot/pull/5423/files#r1523647247) might be the more appropriate fix.

Reply:
See my inline comment.  I’ll try again, but the other options seemed to have different negative outcomes and seemed to move the problem.